src/plots/cartesian/set_convert.js

```diff
@@ -625,14 +625,14 @@ module.exports = function setConvert(ax, fullLayout) {
                     var doesCrossPeriod = false;
 
                     switch(brk.pattern) {
-                        case '%w':
+                        case 'day of week':
```

Comment:
Would be cleaner to pull these out as constants, either at the top of this file or in `cartesian/constants.js`

Reply:
@alexcjohnson would you prefer using the constants in `pattern` def in `layout_attributes.js` as well?

Reply:
Done in https://github.com/plotly/plotly.js/pull/4653/commits/ee2d07af0ce5dd84242cd0738b21b056f296bf6e.

Reply:
> @alexcjohnson would you prefer using the constants in `pattern` def in `layout_attributes.js` as well?

LMK.

Reply:
Sure, might as well make it maximally 🌴

Reply:
Resolving. Done in https://github.com/plotly/plotly.js/pull/4653/commits/db38ce9d38ede3d4a99b1da3ecd73f631833894e.